scripts/extract-latest-release-notes.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * This extracts the latest (non-"Unreleased") notes from a CHANGELOG.md and saves them to `./.tmp/release-notes.md`
+ *
+ * Usage (from project root):
+ * - node scripts/extract-latest-release-notes.js [PATH TO CHANGELOG]
+ * Examples:
+ * - node scripts/extract-latest-release-notes.js ./packages/CHANGELOG.md
+ * - node scripts/extract-latest-release-notes.js ./experimental/CHANGELOG.md
+ */
+
+const fs = require('fs');
+
+const changelog = fs.readFileSync(process.argv[2]).toString();
+const firstReleaseNoteEntryExp = /^## \d+\.\d+\.\d\n.*?(?=^## )/ms;
```

Comment:
Can we add a comment describing the regex for future explorers?

Reply:
Yes, great idea: https://github.com/open-telemetry/opentelemetry-js/pull/4704/commits/cf2f0792abeeac27c999c247efd217b9886fb464 :slightly_smiling_face: